lintly/parsers.py

```diff
@@ -22,6 +22,14 @@ def _normalize_path(self, path):
         """
         Normalizes a file path so that it returns a path relative to the root repo directory.
         """
+        # Override the file/path provided by the tool with what is provided
```

Comment:
To add on to @nandusekarv10 's comment, I'd even go further to say that this entire comment *belongs* in the README (i.e., this comment is optional here).

(Also you have an extra space/indent on the next line.)

Reply:
Added to readme.

Extra space is intentional :).

Reply:
Then two spaces maybe? :) One space is too tiny to indicate 'this is an indent' to me. (Not a big deal regardless, I'm just nit-picking here.)

Reply:
added an extra space